src/opflow/interface/opflowregi.cpp

```diff
@@ -76,10 +76,11 @@ extern PetscErrorCode OPFLOWModelCreate_PBPOLHIOP(OPFLOW);
 
 #if defined(EXAGO_ENABLE_RAJA)
 extern PetscErrorCode OPFLOWModelCreate_PBPOLRAJAHIOP(OPFLOW);
+#endif
```

Comment:
`#ifdef` stil wrong

Reply:
I guess I'm not seeing it. How is it wrong?

Reply:
Let me check the source code directly in your branch...